iocage/lib/ioc_create.py

```diff
@@ -263,9 +264,18 @@ def _create_jail(self, jail_uuid, location):
                         raise RuntimeError(
                             f"RELEASE: {self.release} not found!")
 
-                su.Popen(["zfs", "clone", "-p",
-                          f"{self.pool}/iocage/releases/{self.release}/root@"
-                          f"{jail_uuid}", jail], stdout=su.PIPE).communicate()
+                if not self.thickjail:
+                    su.Popen(["zfs", "clone", "-p",
+                              f"{self.pool}/iocage/releases/{self.release}/root@"
+                              f"{jail_uuid}", jail], stdout=su.PIPE).communicate()
+                else:
+                    su.Popen(["zfs", "create", "-p", jail], stdout=su.PIPE).communicate()
+                    zfs_send = su.Popen(["zfs", "send",
+                              f"{self.pool}/iocage/releases/{self.release}/root@"
+                              f"{jail_uuid}"], stdout=su.PIPE)
+                    su.check_output(["zfs", "receive", "-F", jail], stdin=zfs_send.stdout)
```

Comment:
This can be just `check_call` as we don't care about the output and help reduces memory overhead. We should then catch any `CalledProcessError`s and destroy the parent dataset we made, then inform the user that creation failed via a `logit.exception` call.